python/tests/common.py

```diff
@@ -83,23 +84,24 @@ def _tokenizer(self):
                                  lambda token: G(TypedValue(float(token), 'Number')))
         tokenizer.register_token(r"\d+",
                                  lambda token: G(TypedValue(int(token), 'Number')))
-        tokenizer.register_token(r"'[^']*'",
+        #tokenizer.register_token(r"'[^']*'",
+        #                         lambda token: G(TypedValue(str(token[1:-1]), 'String')))
+        tokenizer.register_token("\"[^']*\"",
```

Comment:
`[^']` should be `[^\"]`
```suggestion
        tokenizer.register_token("\"[^\"]*\"",
```